src/content/structured/components/tabs/code.mdx

```diff
@@ -274,3 +274,65 @@ export const light = [
     <IcTabPanel>Tab Three</IcTabPanel>
   </IcTabContext>
 </ComponentPreview>
+
+### Compact
+
+export const compact = [
+  {
+    language: "Web component",
+    snippet: `<ic-tab-context>
+    <div style={{ width: "60%", margin: "auto" }}>
+  <ic-tab-group label="Example tab group">
+    <ic-tab>One</ic-tab>
+    <ic-tab>Two</ic-tab>
+    <ic-tab>Three</ic-tab>
+    <ic-tab>Four</ic-tab>
+    <ic-tab>Five</ic-tab>
+  </ic-tab-group>
+  <ic-tab-panel>Tab One</ic-tab-panel>
+  <ic-tab-panel>Tab Two</ic-tab-panel>
+  <ic-tab-panel>Tab Three</ic-tab-panel>
+  <ic-tab-panel>Tab Four</ic-tab-panel>
+  <ic-tab-panel>Tab Five</ic-tab-panel>
+  </div>
+</ic-tab-context>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcTabContext>
+    <div style={{ width: "60%", margin: "auto" }}>
```

Comment:
I think this code all needs to be set back a tab, so that it lines up in the react code example. Currently the `</IcTabContext>` doesn't line up with `<IcTabContext>` 
<img width="520" alt="Screenshot 2023-02-21 at 10 45 51" src="https://user-images.githubusercontent.com/111882034/220323954-d438288a-a409-4a4f-97cc-c3b7fef83e4f.png">